test/TimeService.Business.UnitTests/Commands/WorkTime/EditWorkTimeCommandTests.cs

```diff
@@ -1,109 +1,177 @@
 ﻿using FluentValidation;
+using LT.DigitalOffice.Kernel.AccessValidatorEngine.Interfaces;
+using LT.DigitalOffice.Kernel.Exceptions.Models;
 using LT.DigitalOffice.TimeService.Business.Commands.WorkTime;
 using LT.DigitalOffice.TimeService.Business.Commands.WorkTime.Interfaces;
 using LT.DigitalOffice.TimeService.Data.Interfaces;
 using LT.DigitalOffice.TimeService.Mappers.Db.Interfaces;
+using LT.DigitalOffice.TimeService.Mappers.Requests.Interfaces;
 using LT.DigitalOffice.TimeService.Models.Db;
 using LT.DigitalOffice.TimeService.Models.Dto.Requests;
+using LT.DigitalOffice.TimeService.Models.Dto.Requests.HelpersModels;
 using LT.DigitalOffice.TimeService.Validation.Interfaces;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.JsonPatch;
+using Microsoft.AspNetCore.JsonPatch.Operations;
 using Moq;
+using Newtonsoft.Json.Serialization;
 using NUnit.Framework;
 using System;
+using System.Collections.Generic;
 
 namespace LT.DigitalOffice.TimeService.Business.UnitTests.Commands.WorkTime
 {
     public class EditWorkTimeCommandTests
     {
         private Mock<IEditWorkTimeRequestValidator> _validatorMock;
-        private Mock<IDbWorkTimeMapper> _mapperMock;
+        private Mock<IPatchDbWorkTimeMapper> _mapperMock;
         private Mock<IWorkTimeRepository> _repositoryMock;
+        private Mock<IHttpContextAccessor> _httpContextAccessorMock;
+        private Mock<IAccessValidator> _accessValidatorMock;
         private IEditWorkTimeCommand _command;
 
-        private EditWorkTimeRequest _request;
-        private DbWorkTime _editedWorkTime;
+        private JsonPatchDocument<EditWorkTimeRequest> _request;
+        private JsonPatchDocument<DbWorkTime> _dbWorkTimeJsonPatchDocument;
+        private DbWorkTime _editedDbWorkTime;
+        private Dictionary<object, object> _items;
 
         [OneTimeSetUp]
         public void OneTimeSetUp()
         {
-            _request = new EditWorkTimeRequest()
+            _request = new JsonPatchDocument<EditWorkTimeRequest>(new List<Operation<EditWorkTimeRequest>>
+                {
+                    new Operation<EditWorkTimeRequest>(
+                        "replace",
+                        $"/{nameof(EditWorkTimeRequest.Title)}",
```

Comment:
why is the comma highlighted as deleted?

Reply:
I do not know, but it was not only in this pull request was